### PR TITLE
9.0 [MIG] website_odoo_debranding migration

### PR DESCRIPTION
**Pull request by cubells**
- Imported last updates from v8
  - Adapted to v9
  - Renamed to website_odoo_debranding
  - Updated README

@Tecnativa

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - apt-get install ruby-sass
```
Comment (cubells) [inline .travis.yml]: @Yajo 
No Funciona

Comment (yajo) [reply .travis.yml]: All in English here please.
Prefix with `sudo`

**Review by pedrobaeza — APPROVED** [review]
Functional review correct.

This is the aspect before the module:

seleccion_001

and this one is with the module:

seleccion_002

Waiting for Travis result

**Comment by pedrobaeza** [comment]
@lasley, the module you made test for (website_logo) is failing here (https://travis-ci.org/OCA/website/jobs/176021283#L997). Do you figure out why?

**Comment by lasley** [comment]
On it.

FYI this migration is actually for `website_odoo_debranding` not `website_disable_odoo` as the title suggested. I changed it; not sure if that was out of line or anything but it was going to cause some confusion otherwise.

**Comment by pedrobaeza** [comment]
Yeah, the module was renamed, and the other is the old name. No problem in changing it.

**Comment by lasley** [comment]
I tried to review and can't find the old module either though, which is what's causing me all the confusion 😦 . It's possible I need more coffee - still early on my side of the world

**Comment by pedrobaeza** [comment]
It's located in Antiun branch: https://github.com/Antiun/antiun-odoo-addons/tree/8.0/website_disable_odoo

File: website_odoo_debranding/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: AGPLv3 License
+
+===================================
+Remove odoo.com bindings on website
```
Comment (lasley) [inline website_odoo_debranding/README.rst]: I'd say `Remove Odoo Branding from Website` instead, just so we can cover the more abstract - particularly given the module rename

File: website_odoo_debranding/README.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+===================================
+
+This module remove odoo.com bindings on website:
```
Comment (lasley) [inline website_odoo_debranding/README.rst]: IMO - s/odoo.com bindings/Odoo branding

File: website_odoo_debranding/static/src/js/disable_odoo.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
+/**
+ * # -*- encoding: utf-8 -*-
+ * ##############################################################################
```
Comment (lasley) [inline website_odoo_debranding/static/src/js/disable_odoo.js]: Please use [shorter header](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3). Also the C/Python style `#` should be removed in favor of JS comment style.

File: website_odoo_debranding/static/src/js/disable_odoo.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+ * #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ * #
+ * ##############################################################################
+ */
+
+// Check jQuery available
+if (typeof jQuery === 'undefined') { throw new Error('Theme JavaScript requires jQuery') }
+
+// Check jQuery UI available
+// if (typeof jQuery.ui === 'undefined') { throw new Error('Theme JavaScript requires jQuery UI') }
```
Comment (lasley) [inline website_odoo_debranding/static/src/js/disable_odoo.js]: Please remove unused code

File: website_odoo_debranding/static/src/js/disable_odoo.js [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+// Check Bootstrap JS plugins available
+// if (typeof jQuery.fn.alert === 'undefined') { throw new Error('Theme JavaScript requires Bootstrap Alert plugin') }
+
+// Check Backbone available
+// if (typeof Backbone === 'undefined') { throw new Error('Theme JavaScript requires Backbone') }
+
+
++function ($) {
```
Comment (lasley) [inline website_odoo_debranding/static/src/js/disable_odoo.js]: Is it not possible to use the new requirements system here?

File: website_odoo_debranding/static/src/js/disable_odoo.js [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    'use strict';
+
+    // LIBRARY OR OBJECT CODE HERE
+
+
+    // Called when the HTML-Document is loaded and the DOM is ready, even if all the graphics haven’t loaded yet
+    $(document).ready(function() {
+
+        // CODE HERE
```
Comment (lasley) [inline website_odoo_debranding/static/src/js/disable_odoo.js]: I don't see any code being used here. Is this file necessary?

File: website_odoo_debranding/static/src/xml/disable_odoo.xml [hunk 8]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
```
Comment (lasley) [inline website_odoo_debranding/static/src/xml/disable_odoo.xml]: Please remove this file

File: website_odoo_debranding/views/disable_odoo.xml [hunk 9]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <!-- Remove "Create a free website with Odoo" -->
+    <template id="layout_footer_copyright"
+              inherit_id="website.layout_footer_copyright"
+              name="Remove footer Odoo link" priority="20">
+        <xpath expr="//div[@class='pull-right']" position="replace">
+            <div> </div>
```
Comment (lasley) [inline website_odoo_debranding/views/disable_odoo.xml]: IMO you should also add the same `pull-right` class to it that the original div in order to maintain some compatibility with modules that may be looking for this div.

**Review by lasley — CHANGES_REQUESTED** [review]
Thanks @cubells

**Review by lasley — APPROVED** [review]
Thanks @cubells 

Please squash the commits of the same users into each other & your rebase on current 9.0 to get the fix for the Travis SASS errors. We should be good to merge then 😄
